From: David Herrmann <dh.herrmann@gmail.com>
Date: Fri, 18 Jul 2014 12:58:00 +0200
Subject: journal: reduce test-journal-send timeout from 10s to 1s

The sleep(10) in test-journal-send is quite aggressive. We need it only
for the journal to get our cgroup information. But even that information
is not vital to the test, so a sleep(1) should be just fine.

(cherry picked from commit 037ee337f0f64bd35ced765f2e2d97f496d4e7c7)
---
 src/journal/test-journal-send.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/journal/test-journal-send.c b/src/journal/test-journal-send.c
index 3e986ed..45eb327 100644
--- a/src/journal/test-journal-send.c
+++ b/src/journal/test-journal-send.c
@@ -72,7 +72,7 @@ int main(int argc, char *argv[]) {
                         "N_CPUS=%li", sysconf(_SC_NPROCESSORS_ONLN),
                         NULL);
 
-        sleep(10);
+        sleep(1);
 
         return 0;
 }
